docs/content/patterns/conditional-reveal/index.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: Conditionally revealed questions
+group: Patterns
+description: Conditionally reveal a related question when a user selects a specific radio or checkbox option. This ensures that users only encounter questions when it is applicable to their selection.
+version: 1.0.0
+---
+
+<DoHeading />
+
+- only reveal 1 related question
+- use another page if there are multiple related questions
+- only use with vertically stacked checkboxes or radios
```

Comment:
Do we require that the conditional field appears directly after the field that triggers it?

What about a combination of factors causing a field to be revealed?

Reply:
I've added a new do "reveal the related question directly under the initial question" which I think covers this.